docs/source/DeveloperGuide/AddingNewComponents/TestingAgents/index.rst

```diff
@@ -22,12 +22,12 @@ So, what can be wrote down are integration tests::
    class MyAgentTestCase(unittest.TestCase):
```

Comment:
It seems that in this example there are different deviations.
Maybe it's better to use `pytest`?

Reply:
All these examples of unit tests should be better re-written using pytest, but yesterday I did not have time for it.